src/linter/html/transpiler.ts

```diff
@@ -161,6 +202,45 @@ function lintBootstrapAttributes(tag: Tag, report: HtmlReporter) {
 	}
 }
 
+function checkPreloadAttr(attr: Attribute, report: HtmlReporter) {
+	const value = attr.value.value.toLowerCase();
+	if (value && !["auto", "async", "sync"].includes(value)) {
+		report.addMessage(MESSAGE.REDUNDANT_BOOTSTRAP_PARAM_ERROR, {
+			name: attr.name.value,
+			messageDetails: "Use sap-ui-debug=true to suppress library preload requests",
+		}, attr.name);
+	}
+}
+
+function checkOriginInfoAttr(attr: Attribute, report: HtmlReporter) {
+	if (attr.value.value.toLowerCase() === "true") {
+		report.addMessage(MESSAGE.ABANDONED_BOOTSTRAP_PARAM_ERROR, {
+			name: attr.name.value,
+		}, attr.name);
+	} else {
+		report.addMessage(MESSAGE.ABANDONED_BOOTSTRAP_PARAM, {
+			name: attr.name.value,
+		}, attr.name);
+	}
+}
+
+function checkBindingSyntaxAttr(attr: Attribute, report: HtmlReporter) {
+	if (attr.value.value.toLowerCase() === "complex") {
+		report.addMessage(MESSAGE.REDUNDANT_BOOTSTRAP_PARAM, {
+			name: attr.name.value,
+			messageDetails: "Only 'complex' is supported with UI5 2.X and automatically" +
+				" enforced by the UI5 runtime. Complex binding parser supports simple binding syntax per default.",
+		}, attr.name);
+	} else {
+		report.addMessage(MESSAGE.REDUNDANT_BOOTSTRAP_PARAM_ERROR, {
+			name: attr.name.value,
+			messageDetails: "Only 'complex' is supported with UI5 2.X and automatically" +
+				" enforced by the UI5 runtime. Check all bindings whether they will be " +
+				"misinterpreted in 2.0 with binding syntax 'complex'.",
```

Comment:
2.0 vs 2.x.

I think it should be consistent, right?

Reply:
Thanks!